### DISCUSSION TITLE
Release ?

### DISCUSSION THREAD

**Discussion by lukas-cc** [lookfirst/mui-rff#1176] [General]
Just wanted to ask if you could do a release soon with the latest MUI 6 changes ? Thank you.

**Comment by lookfirst** [comment]
I am waiting for parcel to fix their stuff.

https://github.com/parcel-bundler/parcel/issues/9926#issuecomment-2369971853

**Comment by lukas-cc** [comment]
I see makes sense. There seems to be some progress: https://github.com/nodejs/node/issues/54573

**Comment by lookfirst** [comment]
Great, lmk when it works and I'll do a release. =)

**Comment by lukas-cc** [comment]
I think it should be fixed now in node 22.10 or 23.

**Comment by lookfirst** [comment]
I run node from brew... this is all I got right now... node 22.9.0 -> 22.9.0_1

Once it is upgraded, I'll be sure to release. =)

**Comment by lookfirst** [comment]
==> Upgrading node
  22.9.0_1 -> 23.1.0 
==> Pouring node--23.1.0.arm64_sequoia.bottle.tar.gz

I just noticed that node got upgraded...

**Comment by lookfirst** [comment]
I've just published v8.0.1. Thanks all for your patience!

**Reply by lukas-cc** [reply]
Thanks!
